<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GreetingPage</class>
 <widget class="QMainWindow" name="GreetingPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>503</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GreetingPage</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="CheckAccount">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>100</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Already have an account?</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>60</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Let's meet new people</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnRegister">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>360</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Register</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>150</y>
      <width>141</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="Login">
       <property name="text">
        <string>login</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="login"/>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="RegLabel">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>320</y>
      <width>291</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Don't have an account? Let's register!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnLogIn">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>220</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Log in</string>
    </property>
   </widget>
   <widget class="QLabel" name="Greeting">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>321</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Welcome to the HobbyHub, my friend!</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>180</y>
      <width>141</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="Password">
       <property name="text">
        <string>password</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="password"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>270</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="forgotlabel">
       <property name="text">
        <string>Forgot password?</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BtnForgottenPass">
       <property name="text">
        <string>Forgot</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>layoutWidget</zorder>
   <zorder>CheckAccount</zorder>
   <zorder>Label</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>RegLabel</zorder>
   <zorder>BtnLogIn</zorder>
   <zorder>Greeting</zorder>
   <zorder>layoutWidget_2</zorder>
   <zorder>BtnRegister</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>503</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
